### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server-Default Documents_Screen Reader: NVDA is reading "Clickable File Name" but on pressing enter no action is happening inside the table.

### ISSUE THREAD

**Issue by Shayanti123** [Microsoft/IIS.WebManager#110]
**Environment Details:**
URL:https://manage.iis.net/webserver/certificates
Browser Details: Google Chrome Version 68.0.3440.106
Screen Reader: NVDA
**MAS ID Violated:** MAS1.3.1, MAS2.1.1

**Repro Steps:**
1) Download and install "Microsoft IIS Administration" application.
2) Enable Internet information Services.
3) Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4) Tab till Web Server and press enter
5) Tab till Default Documents and press enter
6) Start Screen Reader (Narrator/NVDA).
7) Verify screen reader is reading all the contents, controls,links and images properly throughout the process.

**Actual Result:**
On pressing down arrow, NVDA is reading Clickable File Name but when the user press Enter no action is happening inside the table.
And also the File Name and the Sort button(up arrow) is inaccessible through Keyboard though the control is interactive through Mouse

**Note:**
Similar issue observed in pages where Sort button exists
 
**Expected Result:**
Screen Reader should read all the contents, controls, links and images properly throughout the process.

**User Impact:** 
Visually Impaired Users might face difficulty in navigating through the contents and controls if screen reader does nor read the controls properly

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachments:**
[Bug_110_On pressing Enter or Space Bar on File Name there is no action happening inside the table.pptx](https://github.com/Microsoft/IIS.WebManager/files/2323884/Bug_110_On.pressing.Enter.or.Space.Bar.on.File.Name.there.is.no.action.happening.inside.the.table.pptx)

![bug_110_the file name and sort button is inaccessible through the keyboard](https://user-images.githubusercontent.com/42638915/44703753-6e885b80-aab6-11e8-9db3-a586f1e33284.jpg)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #ScreenReader; #MAS1.3.1; #C+E-H; #.NETWebsites; #Chrome; #NVDA; #MAS2.1.1; #Keyboard

**Comment by Roopesh-T** [comment]
**GitTags**: #A11ySev3;
